src/consensus/raft/raftconsensus.h

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "kv/kvtypes.h"
+#include "raft.h"
+
+#include <memory>
+
+namespace raft
+{
+  template <class LedgerProxy, class ChannelProxy>
+  class RaftConsensus : public kv::Consensus
```

Comment:
I think this adapter makes things much more readable. Can we add a brief doc comment on this `RaftConsensus` class explaining its simply mapping between generic and Raft terminology, permitting a plausibly independent standalone Raft?